README.md

```diff
@@ -29,30 +29,41 @@ We have several demo apps to help provide reference for how to use the SDK:
 
 4. On the first build: Enable & Trust the OpenAPIGenerator extension when prompted.
 
-5. The quickest way to try out the demo for remote inference is using Together.ai's Llama Stack distro at https://llama-stack.together.ai - you can skip Step 6 unless you want to build your own distro. 
+5. The quickest way to try out the demo for remote inference is using Together.ai's Llama Stack distro at https://llama-stack.together.ai - you can skip Step 6 unless you want to build your own distro. Note that Llama 4 is currently supported by building your own distro from Llama Stack main. This can be done by:
+
```

Comment:
Can we mention the GPU requirements to build own distro to give people a headsup?